less/home-page.less

```diff
@@ -53,9 +147,42 @@
     display: flex;
     justify-content: space-between;
 
+	@media @mobile {
+		width: 100%;
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+		justify-content: center;
+	}
+
     .destination {
       width: 30%;
       margin-bottom: 30px;
+
+	  @media @mobile {
+		  width: 100%;
+		  text-align: center;
+		  display: flex;
+		  flex-direction: column;
+		  justify-content: center;
+		  align-items: center;
+	  }
+
     }
   }
-}// home
\ No newline at end of file
+}// home
+
+//button for content-pick section
+
+.btn {
+	.custom-btn(200px, 45px, @button-bg);
+	text-align: center;
+	.custom-layout(center, center);
+	color: @white;
+
+	@media @mobile {
+		width: 70%;
+
+	}
+
+}
```

Comment:
Nice media queries. Awesome job at keeping at constant tab spacing I would suggest and use one tab space though so that your code is closer together and quicker to read